pkg/webhook/webhook.go

```diff
@@ -19,17 +19,18 @@ import (
 	"net/http"
 	"os"
 
+	"github.com/hashicorp/go-retryablehttp"
 	"github.com/maksim-paskal/aks-node-termination-handler/pkg/config"
 	"github.com/maksim-paskal/aks-node-termination-handler/pkg/template"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 )
 
-var client = &http.Client{}
+var client = &retryablehttp.Client{}
 
-var errHTTPNotOK = errors.New("http result not OK")
+var ErrHTTPNotOK = errors.New("http result not OK")
```

Comment:
I don't see the need to make these changes. This variable used only in this package.

Reply:
You are right, forgot to revert those changes once the switch to retryablehhtp was made. Revert done at bf4a487